src/components/v2/Layout/constants.tsx

```diff
@@ -10,37 +10,37 @@ interface IMenuItem {
 export const menuItems: IMenuItem[] = [
   {
     href: '/dashboard',
-    text: 'Dashboard',
+    text: 'layout.menuItems.dashboard',
```

Comment:
I think text should remain being the actual content rather than a translation key:
```typescript
text: t('layout.menuItems.dashboard')
```